app/services/flat_service.rb

```diff
@@ -19,6 +19,17 @@ def delete_score(score_id)
       conn.delete("/v2/scores/#{score_id}")
     end
 
+    def create_score(title)
+      file = File.read('./xml_files/template.musicxml')
```

Comment:
also not sure where this should live, I'm essentially encoding an empty template, but it might not be the best spot to put it here, just left it for this first pass

Reply:
I think this works for now, but should eventually be moved to its own (probably private) method. Would advocate figuring out whether we need access to this anywhere else before moving.